Source/WebGPU/WGSL/TypeDeclarations.rb

```diff
@@ -164,3 +164,20 @@
         [T < Float, N].(Vector[T, N]) => Vector[T, N],
     }
 end
+
+# 17.3. Logical Built-in Functions (https://www.w3.org/TR/WGSL/#logical-builtin-functions)
+
+# 17.3.1 & 17.3.2
+["all", "any"].each do |op|
+    operator :"#{op}", {
```

Comment:
Q: "all" and "any" are strings and then we interpolate them in `:"#{op}". Is there a way to avoid the string interpolation? I couldn't make it work when I tried but my knowledge of Ruby is only at an intermediate level.

Reply:
My first thought is that we can tweak `operator` to accept either a string or a symbol (it might even work today, but not sure). A quick google also suggests that we can use `op.to_sym`. I'll land this as is, and refactor all the use cases later.